whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -14956,14 +14956,21 @@
       "$f": {"addProperty": "marc:citySubsection", "TODO:ignored": "true?"},
       "$g": {"addProperty": "marc:otherNonjurisdictionalGeographicRegionAndFeature"},
       "$h": {"addProperty": "marc:extraterrestrialArea"},
+      "$0": {"addProperty": "marc:recordControlNumber"},
+      "$1": null,
       "$2": {"property": "marc:sourceOfHeadingOrTerm"},
-      "$0": {"addProperty": "marc:uri"}
+      "$4": {"addProperty": "marc:relatorCode"},
+      "$6": {"property": "marc:fieldref"}
     },
     "753": {
```

Comment:
I'd like to see if we can possibly capture $0 or $1 here, unless more than one of $a, $b and $c is present at the same time (or just put the URI in one of them).

Reply:
I have left a note for $0 until we can take care of it properly, it is the only one which is in LC examples and has a link to a possible vocabulary.